### DISCUSSION TITLE
GPT4 não está disponível?

### DISCUSSION THREAD

**Discussion by Catronis** [betalgo/openai#235] [Q&A]
Hello friends!
Why doesn't it return anything in this code?

var completionResult = await openAiService.ChatCompletion.CreateCompletion(new ChatCompletionCreateRequest
            {
                Messages = Mensagens,
                Model = OpenAI.GPT3.ObjectModels.Models.Gpt_4,
                //MaxTokens = 50//optional
            });

if i change to

            var completionResult = await openAiService.ChatCompletion.CreateCompletion(new ChatCompletionCreateRequest
            {
                Messages = Mensagens,
                Model = OpenAI.GPT3.ObjectModels.Models.ChatGpt3_5Turbo0301,
                //MaxTokens = 50//optional
            });

it works perfectly!

**Comment by kayhantolga** [comment]
Do you have access to GPT_4? It is only available for a limited number of users who received an invitation.

**Reply by Catronis** [reply]
Oh, I didn't know, sorry. I hired the API this week, even for paying people it is not released to everyone?
